servlet/src/test/java/io/undertow/servlet/test/util/DeploymentUtils.java

```diff
@@ -78,6 +98,10 @@ public static Deployment setupServlet(final ServletExtension servletExtension, f
 
     }
 
+    public interface DeploymentConfigurator {
+
+        void accept(DeploymentInfo deploymentInfo);
```

Comment:
Why is this necessary? anything you can do here you can do with a ServletExtension?

Reply:
Updated to use `ServletExtension`.